torch_xla/utils/serialization.py

```diff
@@ -77,11 +77,13 @@ def save(data, path, master_only=True, global_master=False):
   xm.rendezvous('torch_xla.utils.serialization.save')
 
 
-def load(path):
+def load(path, map_location=None):
```

Comment:
per your comment, how do you feel about defaulting the `map_location` to CPU?

Reply:
It would be a breaking change and it would be different to the default behavior of `torch.load`. I wouldn't suggest it